OpenInActionExtension/Media.xcassets/TouchBarBezel.colorset/Contents.json

```diff
@@ -0,0 +1,14 @@
+{
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  },
+  "colors" : [
+    {
+      "idiom" : "mac",
+      "color" : {
+        "reference" : "systemPurpleColor"
```

Comment:
We could use our very own blurple here?

Reply:
Yes, makes total sense 👍  I completely ignore this color. Fixed!